src/main/java/micdoodle8/mods/galacticraft/core/client/gui/screen/GameScreenCelestial.java

```diff
@@ -31,7 +31,7 @@
     private float centreY;
     private float scale;
 
-    private final int lineSegments = 90;;
+    private final int lineSegments = 90;
```

Comment:
maybe it's worth to run global reformat/cleanup/import optimization in other PR, wdyt?

Reply:
It would make the PR unreadable

Reply:
Feel free. There is lots of dead code and such things